caffe2/image/transform_gpu.cu

```diff
@@ -62,4 +61,19 @@ bool TransformOnGPU<uint8_t, float, CUDAContext>(Tensor<CUDAContext>& X, Tensor<
   return true;
 }
 
+template <>
+bool TransformOnGPU<uint8_t, float16, CUDAContext>(Tensor<CUDAContext>& X, Tensor<CUDAContext> *Y, float std, float mean, CUDAContext *context)
```

Comment:
I swear there was a reason I didn't do that before, but I can't remember, and I fixed it this time, so it obviously wasn't too serious :)

Updated.